pipeline/config-test.groovy

```diff
@@ -317,8 +333,8 @@ environments {
                     memory_limit = "1Gi"
                     uwsgi_threads = 2
                     uwsgi_processes = 4
-                    replica_min = 1
-                    replica_max = 1
+                    replica_min = 2
+                    replica_max = 4
```

Comment:
Are we trying to use more, but, smaller pods?

Reply:
Aitchison asked me increase pods to 2 at min, I didnt try to shuffle the cpu and memory yet, wanted to do that for all envts with more pods and play around a bit more in a diff PR